src/components/feedbackFaces.tsx

```diff
@@ -0,0 +1,45 @@
+import React, { useState } from "react";
+import { event } from "@site/src/utils/gtags.client";
+
+// Default export function component
+export default function FeedbackFaces() {
+    const [clickedFace, setClickedFace] = useState<string | null>(null);
+
+    const faces = [
+        { label: "Sad", emoji: "😞" },
+        { label: "Neutral", emoji: "😐" },
+        { label: "Happy", emoji: "😊" },
+    ];
+
+    const handleFeedbackClick = (feedbackType: string) => {
+        setClickedFace(feedbackType);
+
+        const label = `${feedbackType}`;
+
+        event({
+            action: "feedback_click",
+            category: "feedback",
+            label: label,
```

Comment:
Did you want to lowercase the label for consistency?

Reply:
Ah, ty, missed that